buildtest/cli/__init__.py

```diff
@@ -953,6 +983,13 @@ def report_menu(subparsers):
         action="store_true",
         help="Print output in machine readable format",
     )
+    parser_report.add_argument(
+        "--color",
+        type=supported_color,
+        metavar="COLOR",
+        action="append",
```

Comment:
i think the `--color` option should not be appended multiple times meaning you can't do `buildtest rt --color blue --color red`. That being said please remove `action="append"`. The `type=supported_color` will handle this correctly which will be of type `str` that is a valid color

Reply:
Removed